src/xcode/ENA/ENA/Source/Models/Exposure/RiskExposureMetadata.swift

```diff
@@ -29,12 +29,23 @@ struct RiskExposureMetadata: Codable {
 		self.dateChangedComparedToPreviousSubmission = dateChangedComparedToPreviousSubmission
 	}
 
+	init(
+		riskLevel: RiskLevel,
+		riskLevelChangedComparedToPreviousSubmission: Bool,
+		dateChangedComparedToPreviousSubmission: Bool
+	) {
+		self.riskLevel = riskLevel
+		self.riskLevelChangedComparedToPreviousSubmission = riskLevelChangedComparedToPreviousSubmission
+		self.dateChangedComparedToPreviousSubmission = dateChangedComparedToPreviousSubmission
+	}
+	
 	init(from decoder: Decoder) throws {
 		let container = try decoder.container(keyedBy: CodingKeys.self)
 
 		riskLevel = try container.decode(RiskLevel.self, forKey: .riskLevel)
 		riskLevelChangedComparedToPreviousSubmission = try container.decode(Bool.self, forKey: .riskLevelChangedComparedToPreviousSubmission)
-		mostRecentDateAtRiskLevel = try container.decode(Date.self, forKey: .mostRecentDateAtRiskLevel)
+		mostRecentDateAtRiskLevel = try? container.decode(Date.self, forKey: .mostRecentDateAtRiskLevel)
```

Comment:
I see this pattern at several places and don't really like it. On first glance it feels arbitrary when to use optional try and when not. Let's use this one:
```suggestion
		mostRecentDateAtRiskLevel = try container.decodeIfPresent(Date.self, forKey: .mostRecentDateAtRiskLevel)
```